MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include attrs_strict/py.typed
+recursive-include attrs_strict *.pyi
```

Comment:
this and the line below is not required with setuptools-scm that pulls in all SCM included. files https://github.com/bloomberg/attrs-strict/blob/master/pyproject.toml#L4

so this file may be removed entirely

Reply:
so if I understand you correctly, we just move `py.typed` into `setup.cfg` and remove the rest of this file, and somehow the `*.pyi` files will be included automatically?